src/python/pants/backend/docker/rules_integration_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Another naming nit: `docker_build_integration_test.py`? Since that's the functionality it's testing.